q2_vsearch/_cluster_sequences.py

```diff
@@ -114,7 +114,8 @@ def _parse_uc(fh):
                       sample_ids=sample_ids)
 
 
-def dereplicate_sequences(sequences: QIIME1DemuxDirFmt
+def dereplicate_sequences(sequences: QIIME1DemuxDirFmt,
+                          prefix: bool=False
```

Comment:
Can we rename this param to `derep_prefix`, to match the `vsearch`-given name? We have been trying to do that when exposing params on 3rd-party tools like this.